src/main/java/com/exasol/adapter/document/documentnode/avro/JavaObjectNode.java

```diff
@@ -10,6 +10,7 @@
  * {@link DocumentNode} wrapping a java map.
  */
 public class JavaObjectNode implements DocumentObject {
+    private static final long serialVersionUID = 1980967934702336379L;
     /** @serial */
```

Comment:
Please check other classes with /** @ serial */
I think UID can be missing in some other places too

Reply:
done